conn-soup/README.md

```diff
@@ -140,4 +140,5 @@ evolutionSupportEnabled: true
 ### 3.6.0
 
 + Embedded service based on MINA decodes the message as sender during sending.
-  This approach is important for cases in which a pair of messages have the same protocol message type and different structures depending on the direction.
\ No newline at end of file
+  This approach is important for cases in which a pair of messages have the same protocol message type and different structures depending on the direction.
++ Fixed decoding of EndOfSession messages.
```

Comment:
You should add also a version paragraph for these changes because the current version is not 3.6.0